lib/velum/dex/ldap.rb

```diff
@@ -0,0 +1,66 @@
+require "base64"
+
+module Velum
+  # This class offers the integration between ruby and the Saltstack API.
+  module Dex
+    class << self
+      def ldap_connectors_as_pillar
+        ldap_connectors = DexConnectorLdap.all.map do |con|
+          {
+          "type"            => "ldap",
```

Comment:
the body of the block needs to be indented 2 more spaces, and i also was wondering if we could use the 'new' ruby hash syntax here:

```ruby
{
  type: "ldap",
  id:   con.id,
  ...
}
```

the old way `key => value` is generally outdated and only used when there e.g. is a `-` in the key and the parser can't do it, e.g. this doesn't work:

```ruby
{ symbolized-key: "value" }
```

Reply:
I do prefer the new hash syntax, but the rest of velum is written with the old syntax. A quick "grep ' =>' * -RIi | wc -l" across the velum codebase reveals 324 instances of the old syntax.

If we are going to prefer the new style syntax, then I would recommend that all of them be changed simultaneously in a single commit to be rid of the old style everywhere possible.

If we do not want to change them everywhere, why do we want to only use the clean syntax in some places? I feel that it makes the code more confusing if there is a random mixture of both old and new style.